### PR TITLE
2218 - 0

### PR DESCRIPTION
**Pull request by VladikRebko**
## Номер
2218

## Номер задания
0

## Ссылка на видео с демо
https://www.youtube.com/watch?v=7832q4ZG-DU&feature=youtu.be

## Комментарии
Обернул RedMine в докер-контейнер.

### CODE REVIEW

File: 2218/0/1.rb~ [hunk 1]
```diff
@@ -0,0 +1,2 @@
+puts 'Chunky bacon'
+
```
Comment (HolyWalley) [inline 2218/0/1.rb~]: ?)

File: 2218/0/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM ruby:2.4
+RUN apt-get update && apt-get install -y build-essential libpq-dev nodejs
+RUN mkdir /app
+WORKDIR /app
+COPY Gemfile /app/Gemfile
+COPY Gemfile.lock /app/Gemfile.lock
+RUN bundle install
+COPY . /app
+
```
Comment (HolyWalley) [inline 2218/0/Dockerfile]: Unnecessary new line. Correlate with the file 1.rb

File: 2218/0/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    command: [ "sh", "-c", "rm -f /app/tmp/pids/server.pid \
+      && ls \
+      && bundle exec rake db:create \
+      && bundle exec rake db:migrate \
+      && bundle exec rails s -p 3000 -b '0.0.0.0'" ]
+    ports:
+      - "3000:3000"
+  db:
+    image: postgres
+
```
Comment (HolyWalley) [inline 2218/0/docker-compose.yml]: Unnecessary new line. Correlate with the file 1.rb

File: 2218/0/1.rb~ (hunk 1 above)
Comment (VladikRebko) [reply 2218/0/1.rb~]: thx))

Comment (HolyWalley) [reply 2218/0/1.rb~]: why do you need this file?

File: 2218/0/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+version: '3'
+services:
+  redmine:
+    build: .
+    links:
+      - db
+    volumes:
+      - .:/app
+    command: [ "sh", "-c", "rm -f /app/tmp/pids/server.pid \
```
Comment (HolyWalley) [inline 2218/0/docker-compose.yml]: can you explain what is it mean?
why do you need all these lines in command: ?
and what will happen every time you start your container?
Also, what happens to the data if I delete your container?

**Pull request merged**